test/test-template-engine.js

```diff
@@ -482,6 +482,16 @@ describe('html rendering', () => {
       ],
     }));
     assert(container.querySelector('#target').textContent === ':foo-one:');
+    render(container, getTemplate({
+      items: [], // Tests a specific issue we had going from some > none > some.
```

Comment:
Specifically flipping back to the _zero length_ case causes this issue to happen.